port/win/env_win.cc

```diff
@@ -812,7 +817,7 @@ Status WinEnvIO::AreFilesSame(const std::string& first,
   }
 
   if (FileInfo_1.VolumeSerialNumber == FileInfo_2.VolumeSerialNumber) {
-    *res = (0 == memcmp(FileInfo_1.FileId.Identifier, FileInfo_2.FileId.Identifier, 
+    *res = (0 == memcmp(FileInfo_1.FileId.Identifier, FileInfo_2.FileId.Identifier,
```

Comment:
Could you run make format? This line is > 80 chars

Reply:
@anand1976  There are many lines breaks the 80 chars limit in the env_win.cc, and my change was removing the tailing empty space (by editor).
Do I need to fix all the long lines or only touched codes?

Reply:
Formatted the entire file by https://google.github.io/styleguide/cppguide.html